_sections/home-english.md

```diff
@@ -233,8 +255,16 @@ If the CSV or XML files use different names for the choice `name` and `label`, a
 | select_one_from_file households.csv     | hh   | Select household number           | value=housenum, label=housename |
 | ======================================= | ==== | ==================================|=================|
 | survey                                  |      |                                   |                 |
+Caption: CSV and XML usage
+
+| type                                          | name | label                               | appearance   | parameters               |
+| --------------------------------------------- | ---- | ----------------------------------- | ------------ | ------------------------ |
+| select_one_from_file health_facility.geojson  | site | Select the health facility visited  |   maps       | value = id, label = name |
+| ============================================= | ==== | ====================================|==============| ======================== |
+| survey                                        |      |                                     |              |                          |                     
+Caption: GeoJSON usage
 
-Note that this question type is generally the preferred way of building select questions from external data as it is the most versatile and works across applications. However, if your external data file consists of many thousands of lines, please test carefully whether the performance is satisfactory on the lowest spec device you intend to use. If it is too slow, consider using [External Selects](#external-selects) or [Dynamic selects from preloaded data](#dynamic-selects-from-pre-loaded-data) if your data collection application supports it. 
+Note that, this question type is generally the preferred way of building select questions from external data as it is the most versatile and works across applications. However, if your external data file consists of many thousands of lines, please test carefully whether the performance is satisfactory on the lowest spec device you intend to use. If the form is too slow for CSV or XML files, consider using [External Selects](#external-selects) or [Dynamic selects from preloaded data](#dynamic-selects-from-pre-loaded-data) if your data collection application supports it. 
```

Comment:
I think the original sentence better captured the intent! What it's saying is that if this way of attaching external data is slow in a particular context, there are other options.